class.jetpack-gutenberg.php

```diff
@@ -13,53 +13,20 @@
 /**
  * Wrapper function to safely register a gutenberg block type
  *
- * @param string $slug Slug of the block.
- * @param array  $args Arguments that are passed into register_block_type.
+ * @deprecated 9.1.0 Use Automattic\\Jetpack\\Blocks::Blocks::jetpack_register_block instead
```

Comment:
Too many "Blocks" in there.
```suggestion
 * @deprecated 9.1.0 Use Automattic\\Jetpack\\Blocks::jetpack_register_block instead
```